src/meson.build

```diff
@@ -36,7 +36,7 @@ executable(
     install: true)
 install_symlink(
     'netplan',
-    pointing_to: join_paths('..', '..', '..') + join_paths(get_option('prefix'), libexec_netplan, 'generate'),
+    pointing_to: join_paths('..', '..', '..', '..') + join_paths(get_option('prefix'), libexec_netplan, 'generate'),
     install_dir: systemd_generator_dir)
```

Comment:
that's not quite right. as systemd_generator_dir can and is of different length on different platforms.

maybe install_symlink accepts absolute locations which then can be "optimized" automatically, or later on by packaging itself? (i.e. in debian debhelper fixes up absolute symlinks to relative ones) Doing absolute target is better here imho.

Reply:
I'll adjust this to use absolute paths.